src/test/java/org/webjars/WebJarVersionLocatorTest.java

```diff
@@ -54,14 +71,18 @@ class InspectableCache implements WebJarCache {
             @Override
             public Optional<String> computeIfAbsent(String key, Function<String, Optional<String>> function) {
                 Function<String, Optional<String>> inspectableFunction = function.andThen((value) -> {
-                    numLookups.incrementAndGet();
+                    if(shouldInspect.get()) {
+                        numLookups.incrementAndGet();
+                    }
                     return value;
                 });
                 return cache.computeIfAbsent(key, inspectableFunction);
             }
         }
 
         final WebJarVersionLocator webJarVersionLocator = new WebJarVersionLocator(new InspectableCache());
+        // enable inspection after webJarVersionLocator has been constructed, to ignore lookups caused by loading locator.properties
+        shouldInspect.set(true);
```

Comment:
My first Idea was to modifiy the `assertEquals` calls for `numLookups` but I think that this is a better solution.